addons/database-backup-and-maintenance.sh

```diff
@@ -82,6 +82,7 @@ if [ -f /var/lib/mysql/grastate.dat ]; then
     FIRST_SERVER=`mysql -u$REP_USER -p$REP_PWD -e 'show status like "wsrep_incoming_addresses";' | tail -1 | awk '{ print $2 }' | awk -F "," '{ print $1 }' | awk -F ":" '{ print $1 }'`
     if ! ip a | grep $FIRST_SERVER > /dev/null; then
         SHOULD_BACKUP=0
+        echo "Not the first server of the cluster: database backup canceled."
```

Comment:
I think I can add an `exit 0` statement after that line because processing replication block (at end of file) doesn't made sense in that situation.

What's your opinion ?

Reply:
exit 0 will bypass the replication

Reply:
As discussed with @fdurand , we can do an `exit 0` at that point. If we are not the first server of the cluster, we don't have backups to replicate. Moreover, replication block is more related to active-passive clustering and not impact by following test:
```
if [ -f /var/lib/mysql/grastate.dat ]
```

Reply:
Hi